src/Widgets/GroupWidget/GroupWidgetEditingConfig.js

```diff
@@ -0,0 +1,7 @@
+import * as Scrivito from "scrivito";
+import groupWidgetIcon from "../../assets/images/group_widget.svg";
+
+Scrivito.provideEditingConfig("GroupWidget", {
+  title: "Group",
```

Comment:
* [ ] FYI I see we aren't using `description` yet. I wonder if this is a good candidate. As the widget in itself doesn't add anything to the content, we could explain it using `description`.
On the other hand, the widget's simplicity makes it kinda self-explanatory, so feel free to go without description.

Reply:
I think it's good enough.